### PR TITLE
adding gideon, edward, and ben's project

### PR DESCRIPTION
**Pull request by wuedward**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ See [Project Logistics](projects/logistics.md) for important dates and deliverab
 * [Project Deets](projects/project-deets.md) - **Team:** [Michael Wang](people/michael-wang.md), [Vivek Sudarsan](people/vivek-sudarsan.md), [Julian Ferdman](people/julian-ferdman.md)
 * [Social Timeline](projects/social-timeline.md) - **Team:** [Inna Shteinbuk](people/inna-shteinbuk.md), [Jocelyn Kong](people/jocelyn-kong.md)
 * [Wanderluster](projects/wanderluster.md) - **Team:** [Shimiao Huang](people/shimiao-huang.md), [Yanbo Li](people/yanbo-li.md), [Liudan Xiao](people/liudan-xiao.md)
+* [Smart Alarm](projects/smart-alarm.md) - **Team:** [Gideon Glass](people/gideon-glass.md), [Edward Wu](people/edward-wu.md), [Ben]
```
Comment (dblock) [inline README.md:63]: Ben doesn't have a link?

**Comment by wuedward** [comment]
I don't think he has one

**Comment by dblock** [comment]
So who is this Ben person? He has a full name at the very least no? Why doesn't he have a page?

**Comment by wuedward** [comment]
He's on the slack group, but I don't know he doesn't have a page yet. I've asked for his last name, but he has yet to respond on slack.

**Comment by wuedward** [comment]
DB, can I not merge our project until Ben makes his profile? If that is the case, can I push a version with just Gideon and my name? I put a placeholder for Ben to let other groups know that our team has 3 members already.

**Comment by dblock** [comment]
Just remove him for now, and when Ben makes his page he can add himself. Squash the commits and rebase please.

**Pull request closed without merging**